modules/admin_manual/pages/installation/ubuntu_18_04.adoc

```diff
@@ -1,233 +1,196 @@
-= Manual installation of ownCloud on Ubuntu 18.04
-
+= Install ownCloud on Ubuntu 18.04
 :toc: right
+:logrotate-url: https://linux.die.net/man/8/logrotate
+:webserver-user: www-data
+:webserver-group: www-data
+:install-directory: /var/www/owncloud
 
-TIP: Copy/Paste these commands into you terminal/shell
+This is an ultra-short guide to installing ownCloud on a fresh installation of Ubuntu 18.04.
+Run the following commands in your terminal to complete the installation.
 
-== Preparation
-First we have to prepare our server for ownCloud.
+== Prerequisites
 
-=== Prerequisites
-Freshly installed https://www.ubuntu.com/download/server[Ubuntu 18.04] Server with SSH.
+* A fresh install of https://www.ubuntu.com/download/server[Ubuntu 18.04] with SSH enabled.
+
+== Preparation
 
-==== Update OS
+First, ensure that all of the installed packages are entirely up to date.
 
+[source,console,subs="attributes+"]
 ....
-apt update && apt upgrade -y
+sudo apt update && sudo apt upgrade -y
 ....
 
-==== Create helper script
+=== Create the occ Helper Script
 
+Create a helper script to simplify running xref:configuration/server/occ_command.adoc[occ commands].
+
+[source,console,subs="attributes+"]
 ....
 FILE="/usr/local/bin/occ"
 /bin/cat <<EOM >$FILE
 #! /bin/bash
-cd /var/www/owncloud
-sudo -u www-data /usr/bin/php /var/www/owncloud/occ "\$@"
-EOM
 
-chmod +x /usr/local/bin/occ
+cd {install-directory}
```

Comment:
I am not sure we need this as variables. The webserver user will always be www-data as we decided to make the docs "ubuntu only". I am not sure I understand the meaning behind setting a variable for the install directory. Can you explain?

Reply:
The reason for doing so is to avoid writing the same thing multiple times, and potentially introducing bugs or inconsistencies.